public/_posts/_articles/marco-worms/yearn-v3/hi.md

```diff
@@ -0,0 +1,88 @@
+---
+title:  "Yearn Vaults v3"
+image:
+  src: ./image1.jpg
+  width: 900
+  height: 429
+date: '2022-05-03'
+author: Marco Worms
+translator: jengajojo
+---
+
+![](./image1.jpg?w=900&h=429)
+
+जैसा कि हम रोमांचक परिवर्तनों के लिए तत्पर हैं v3 वॉल्ट डेवलपर्स और डीजेन दोनों के लिए ला रहे हैं, मैंने v3 डिज़ाइन टीम का नेतृत्व करने वाले यर्न योगदानकर्ता के साथ बात की। नई सुविधाओं की संख्या के बारे में जानकर मुझे आश्चर्य हुआ! प्रारंभ में, हमारी बातचीत [ERC-4626](https://twitter.com/iearnfinance/status/1511444220850184197) मानक के आसपास शुरू हुई, जो प्रोटोकॉल में वॉल्ट को मानकीकृत करता है लेकिन यह पता चला है कि यह सिर्फ हिमशैल का सिरा है।
+
+# v3 का उद्देश्य यर्न वॉल्ट के सभी मुख्य पहलुओं में सुधार करना है
+
+### सुरक्षा
+
+डेफी सुरक्षा प्रथाओं के लिए यर्न की स्थिति को सर्वश्रेष्ठ श्रेणी में बनाए रखें।
+
+### विकास
+
+रणनीति बनाने के लिए घर्षण कम करें, और तिजोरी के लिए रणनीतियों की एक विस्तृत श्रृंखला की अनुमति दें।
+
+### स्वचालन
+
+फसल के अलावा और अधिक वॉल्ट फ़ंक्शंस को स्वचालित होने दें, १०० % स्वचालित वॉल्ट के सपने के करीब एक कदम आगे बढ़ें।
+
+## v3 स्कोप से निपटने के लिए, डेवलपर्स निम्नलिखित पर काम कर रहे हैं:
+
+### प्रतिरूपकता बढ़ाएँ
+
+कोड को लागू करना जो yVault अनुबंध में एक मजबूत तरीके से सरल है, जो स्मार्ट लॉजिक को मॉड्यूल में रखने की अनुमति देता है जिसे पुनरावृत्त किया जा सकता है।
+
+ERC-4626 मानक को अपनाना जो हमारे वॉल्ट को किसी भी अन्य प्रोटोकॉल के साथ संगत होने की अनुमति देता है जो इसका पालन करते हैं।
+
+### रणनीति में सुधार करें <-> वॉल्ट संबंध
+
+लॉकिंग फंड, जोखिम भरी रणनीतियों और मल्टीचैन रणनीतियों सहित उपज संभावनाओं की एक विस्तृत श्रृंखला से निपटने के लिए अधिक टूल सहित सरल रणनीति परीक्षण, तैनाती और समग्र रूप से अनुमति दें।
+
+### नए उत्पाद बनाएं
+
+जूनियर ट्रेंच जैसे नए उत्पाद बनाएं जो जोखिम भरी रणनीतियों को सक्षम बनाता है और उपयोगकर्ताओं को उच्च-जोखिम जमा विकल्प चुनने की अनुमति देता है।
+
+प्रत्येक टोकन प्रकार के लिए 1 से अधिक तिजोरी की अनुमति दें
+
+वॉल्ट को मल्टीसिग द्वारा नियंत्रित करने की अनुमति दें जिसे पार्टनर प्रोटोकॉल के साथ साझा किया जा सकता है
+
+## लचीलापन
+
+v3 डेवलपर्स और रणनीतिकारों को उत्पादन के लिए शिपिंग कोड के लिए अधिक लचीलापन देगा। यह अनुबंधों के मॉडर्नाइजेशन को बढ़ाता है और तिजोरियों में जमा करने के नए तरीकों के लिए दरवाजे भी खोलता है।
+
+हम परिधीय अनुबंधों का एक बेहतर स्वचालितकरण भी देखेंगे: v2 में केवल कटाई कार्य स्वचालित होते हैं और, v3 में हम १००% स्वचालित वॉल्ट के करीब पहुंचेंगे क्योंकि अब [Keep3r नेटवर्क] के साथ अधिक महत्वपूर्ण कार्यों को स्वचालित करना आसान है। (https://keep3r.network/)।
```

Comment:
link text and url are not formatted properly [link_text](link_url).
This is the exact same error I get copy pasting google translate. 
![image](https://user-images.githubusercontent.com/32738380/169541462-94e05ee8-56c8-4961-a5b8-ec464f115ea6.png)